include/odp/api/spec/event.h

```diff
@@ -209,6 +209,40 @@ void odp_event_free_multi(const odp_event_t event[], int num);
  */
 void odp_event_free_sp(const odp_event_t event[], int num);
 
+/**
+ * Event flow id value
+ *
+ * Returns the flow id value set in the event.
+ * Flow id value set in the event is used by scheduler to distribute the event
+ * across different flows. For packets received through packet input initial
+ * event flow id will be same as flow hash generated for packets. The hash
+ * algorithm and the header fields defining the flow (therefore used for
+ * hashing) is platform dependent. For all other events initial flow id is zero
```

Comment:
Not accurate: "The hash algorithm and the header fields defining the flow (therefore used for hashing) is platform dependent."

Hash algorithm and thus flow id value is implementation specific, but which header fields that are included into hash calculation are not (pktio config spec includes the options already).

So, replace with e.g. this:
"The hash algorithm and therefore the resulting flow id value is implementation specific. Use pktio API configuration options to select the fields used for initial flow id calculation."